src/Sepes.Functions/SelfMonitoringFunction.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Text;
+using Microsoft.ApplicationInsights;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+
+namespace Sepes.Functions
+{
+    public static class SelfMonitoringFunction
+    {
+        [FunctionName("SelfMonitoring")]
+        public static void Run([TimerTrigger("0 */15 * * * *")]TimerInfo myTimer, ILogger log)
```

Comment:
I forgot the convention, how often is this?

Reply:
Every 15 mins. Do you think it's to often?